KTXDeveloperGuideWebGL.md

```diff
@@ -0,0 +1,42 @@
+# KTX 2.0 / Basis Universal Textures — WebGL Tips
+
+To ensure that transcoding from the transmission texture formats (i.e. ETC1S or UASTC) to GPU compressed formats works as expected, developers should test their WebGL applications on a wide range of platforms due to differences in compressed format support. Still, there are two setups that provide exceptionally high level of compatibility: modern Intel GPUs (Windows and Linux) and Apple M1 SoC (macOS).
+
+> **Note**: At the time of writing (March 2021), Chromium Dev builds (v.91) should be used as the latest texture-related updates are not in the stable branch yet.
+
+## Windows and Linux
+
+Modern Intel GPUs, namely Intel HD Graphics 5xx ("Skylake") or newer, have the best compressed format coverage on Windows and Linux. OpenGL or Vulkan are required to get access to ASTC and ETC formats.
+
+1. Consider using the latest GPU driver. On Windows, it could be downloaded from the [Intel website](https://downloadcenter.intel.com/); on Linux, it is usually provided and maintained by OS distribution vendors.
+
+2. Install [Google Chrome Dev](https://www.google.com/chrome/dev/) or [Microsoft Edge Dev](https://www.microsoftedgeinsider.com/en-us/download/).
+
+3. On Windows, open `chrome://flags/#use-angle` page, select OpenGL as ANGLE graphics backend, and restart the browser.
+
+4. Confirm enabled WebGL extensions here: https://webglreport.com/.
+
+## macOS
+
+Apple M1 SoC supports all possible compressed transcode targets: BC*, ETC*, ASTC, and PVRTC1. Most of them are available only via Metal API. Intel-based Macs support only BC* formats.
+
+1. Install [Google Chrome Dev](https://www.google.com/chrome/dev/) or [Microsoft Edge Dev](https://www.microsoftedgeinsider.com/en-us/download/).
+
+2. Run it with the following cmd-line parameters: `--use-cmd-decoder=passthrough --use-gl=angle --use-angle=metal`.
+
+3. Confirm enabled WebGL extensions here: https://webglreport.com/.
+
+## Platform Support Table
```

Comment:
Might be helpful to include the relevant WebGL extensions in the table, if they'll fit. Or just link to https://github.com/KhronosGroup/3D-Formats-Guidelines/blob/main/KTXDeveloperGuide.md#gpu-api-support?

Reply:
I'll add the extension names to the steps above.